src/libs/OptionsListUtils.ts

```diff
@@ -23,41 +34,138 @@ import * as TaskUtils from './TaskUtils';
 import * as TransactionUtils from './TransactionUtils';
 import * as UserUtils from './UserUtils';
 
+type Tag = {
+    enabled: boolean;
+    name: string;
+    accountID: number | null;
+};
+
+type Option = Partial<ReportUtils.OptionData>;
+
+type PayeePersonalDetails = {
+    text: string;
+    alternateText: string;
+    icons: OnyxCommon.Icon[];
+    descriptiveText: string;
+    login: string;
+    accountID: number;
+};
+
+type CategorySection = {
+    title: string | undefined;
+    shouldShow: boolean;
+    indexOffset: number;
+    data: Option[];
+};
+
+type Category = {
```

Comment:
Why we can't use `PolicyCategory` instead of this type?

Reply:
I think this is something different in this file we are only setting those object as `{name: string, enabled: boolean}` like here for example 
![image](https://github.com/Expensify/App/assets/25230417/b2ba5827-a7f7-427c-95cc-ec2123dab245)